include/string.h

```diff
@@ -50,6 +50,7 @@ char* __fastcall__ strcpy (char* dest, const char* src);
 size_t __fastcall__ strcspn (const char* s1, const char* s2);
 char* __fastcall__ strerror (int errcode);
 size_t __fastcall__ strlen (const char* s);
+size_t __fastcall__ strnlen (const char* s, size_t maxlen);
```

Comment:
@polluks is right.  Please move this line down to the "Non-standard" section of the header.

Reply:
It's conforming to POSIX.1-2008. Still sure?